Digizuite/Metadata/MetaFieldService.cs

```diff
@@ -41,7 +44,19 @@ public class MetaFieldService : IMetaFieldService
                 _logger.LogError("Failed to load metafields", nameof(response), response);
                 throw new Exception("Failed to load metafields: " + response);
             }
-            
+
+            var languageId = await _damAuthenticationService.GetLanguageId();
```

Comment:
Should this only be done if there is no data in those properties? Seems like you are always doing it now regardless of what DAM version we are targeting

Reply:
yes, but that should not affect performance, since accesskey is cached

Reply:
I am not concerned with performance, more the fact that we may overload it with something that is not right - all thought maybe that is not happening?

Reply:
Case we discussed in DM will never happen, since in 5.6 we return only labels array and LabelId/Label/LanguageId  do not have values

https://github.com/Digizuite/digizuite.core/blob/master/Apps/LegacyService/Services/MetaFieldConverter.cs#L100

and in 5.5.* we return only values with LabelId/Label/LanguageId and there is no Labels array

https://github.com/Digizuite/digizuite.core/blob/release/5.5.3/Apps/LegacyService/Services/MetaFieldResponseConverter.cs#L79